docs/xdr/features/collect/ingestion_methods/sekoiaio_docker_concentrator.md

```diff
@@ -0,0 +1,435 @@
+
+#### Additional options
+
+```yaml
+restart: always
+pull_policy: always
+```
+
+* `restart: always`: this line indicates to restart the concentrator everytime it stops. That means if it crashes, if you restart Docker or if you restart the host, the concentrator will start automatically.
+* `pull_policy: always`: docker compose will always try to pull the image from the registry and check if a new version is available for the tag specified.
+
+## Start the concentrator
+
+To start the concentrator, run the following command in the folder where `docker-compose.yml` and `intakes.yaml` are stored:
+
+```bash
+sudo docker compose up -d
+```
+
+You should have have an answer like this:
+
+```bash
+[+] Running 1/1
+ ⠿ Container docker-compose-rsyslog-1  Started
+```
+
+!!! Note
+    The first time you start the container, you will download the image stored on the GitHub repository - `ghcr.io/sekoia-io/sekoiaio-docker-concentrator`
+
+To check the status of the concentrator, you can run:
+
+```bash
+sudo docker compose ps
+```
+
+## Useful commands
+
+**To view all the concentrator logs:**
+
+```bash
+sudo docker compose logs
+```
+
+Everytime an event is sent to the concentrator, the event is shown in these logs.
+
+**To view last logs in real time:**
+
+```bash
+sudo docker compose logs -f
+```
+
+**To stop the concentrator:**
+
+```bash
+sudo docker compose stop
+```
+
+**To delete the concentrator (the concentrator needs to be stopped):**
+
+```bash
+sudo docker compose rm
+```
+
+## Troubleshooting
+
+You can't find the logs in your community? No worries this section will give you an advice to identify what is happening.
+
+### Step 1: check if the events are received by the concentrator
+
+To check if the events are received by the concentrator, you can run the following command that will display all the last logs received:
+
+```bash
+sudo docker compose logs
+```
+
+If you are looking for specific logs, you can also run:
+
+```bash
+sudo docker compose logs | grep "YOUR_PATTERN"
+```
+
+Finally, if you want to check if events are coming in real time:
+
+```bash
+sudo docker compose logs -f
+```
+
+**You don't see your events with these commands?** 
+
+1. Check the `intakes.yaml` file to see if you have declared the protocols and ports you wanted. 
+
+2. Verify if this information is taken into account by the concentrator. At start-up, the concentrator always shows the list of Intakes with the protocols and ports.
+    ```bash
+    sudo docker compose logs | more
+    ```
+
+3. Check that you correctly declared the `ports` section in the `docker-compose.yml` file. They MUST be the sames as the ports declared in the `intakes.yaml` file. For instance, if you declared 4 technologies on ports `25020`, `25021`, `25022` and `25023`, the ports line the `docker-compose.yml` has to be `"25020-25023:25020-25023"`. 

```

Comment:
3. Check that you correctly declared the `ports` section in the `docker-compose.yml` file. They MUST be the same as the ports declared in the `intakes.yaml` file. For instance, if you declared 4 technologies on ports `25020`, `25021`, `25022` and `25023`, the ports line the `docker-compose.yml` has to be `"25020-25023:25020-25023"`.